PowerShell.Common.props

```diff
@@ -104,6 +94,9 @@
 
     </PropertyGroup>
 
+    <!-- Remove existing generated files by source generators because new source code will soon be generated when the build starts -->
+    <RemoveDir Directories="gen\SourceGenerated" Condition="Exists('gen\SourceGenerated')" />
```

Comment:
The path `gen\SourceGenerated` is duplicated in here and `System.Management.Automation.csproj`. I tried creating a property holding the path in the `PropertyGroup` right above, and use it in `RemoveDir` and `CompilerGeneratedFilesOutputPath`. However, it works in `RemoveDir`, but not in `CompilerGeneratedFilesOutputPath`, and I cannot figure out why. I'd love if someone can make it work.